### PR TITLE
Release 2.9.2: Release branch

### PR DESCRIPTION
**Pull request by sb230132**
100901bb Fix repartition behavior when attaching data node
b16cac4e Fix ChunkAppend, ConstraintAwareAppend child subplan
a7bee6f9 Fix default data node availability status
d74b42ce Fix CAgg on CAgg variable bucket size validation
707acf4b Fix issue with deleting data node and dropping database
6113a7ed Fix CAgg on CAgg different column order
12c13147 Fix adding column with NULL constraint
e93f4ea1 Fix telemetry_stats test failure in PG15
f0481191 Remove parallel safe from partialize_agg

### CODE REVIEW

File: tsl/test/expected/cagg_on_cagg_dist_ht.out [hunk 1]
```diff
@@ -37,6 +37,11 @@ GRANT USAGE ON FOREIGN SERVER :DATA_NODE_1, :DATA_NODE_2, :DATA_NODE_3 TO PUBLIC
 GRANT CREATE ON SCHEMA public TO :ROLE_DEFAULT_PERM_USER;
 -- Global test variables
 \set IS_DISTRIBUTED TRUE
+<<<<<<< HEAD
```
Comment (pallavisontakke) [inline tsl/test/expected/cagg_on_cagg_dist_ht.out:40]: These lines' merge might need correction.

**Review by jnidzwetzki — APPROVED** [review]

**Review by pmwkaa — APPROVED** [review]

**Comment by nikkhils** [comment]
It's very important to get commit https://github.com/timescale/timescaledb/commit/4118a72575d4a1d493ec0884e659e6a7437fd3a0 "Remove parallel safe from partialize_agg" into 2.9.2 release.

**Comment by sb230132** [comment]
> It's very important to get commit [4118a72](https://github.com/timescale/timescaledb/commit/4118a72575d4a1d493ec0884e659e6a7437fd3a0) "Remove parallel safe from partialize_agg" into 2.9.2 release.

Thanks. Fixed it by cherry picking @fabriziomello PR

**Review by pallavisontakke — APPROVED** [review]
Changes look good to me.

**Pull request merged**